client/config/firebaseSetup.ts

```diff
@@ -0,0 +1,17 @@
+import { initializeApp } from 'firebase/app';
+import { getAuth } from 'firebase/auth';
+
+// Linkta web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: 'AIzaSyC8E8SFfC-coXYSjFiqoDdtFf_0imHvMrY',
+  authDomain: 'linkta-prod.firebaseapp.com',
+  projectId: 'linkta-prod',
+  storageBucket: 'linkta-prod.appspot.com',
+  messagingSenderId: '53706722050',
+  appId: '1:53706722050:web:0838b30c53273791a1501b',
+};
+
+// Initialize Firebase
+const firebaseApp = initializeApp(firebaseConfig);
```

Comment:
Remember to export to firebase app

Reply:
I didn't export firebaseApp, instead I exported auth because auth is ultimately the service we will be using for authentication purposes. Just trying to reduce the repetitive import statement for files like logIn and signup. Is there a reason why you think we should export firebaseApp instead?